packages/dx-react-scheduler-material-ui/src/templates/layouts/vertical-view-layout.jsx

```diff
@@ -10,14 +10,20 @@ const styles = theme => ({
     overflowY: 'auto',
     position: 'relative',
   },
-  stickyHeader: {
-    top: 0,
-    zIndex: 2,
+  stickyElement: {
     tableLayout: 'fixed',
     position: 'sticky',
     overflow: 'visible',
     background: theme.palette.background.paper,
   },
+  stickyHeader: {
+    top: 0,
+    zIndex: 2,
+  },
+  stickyScale: {
+    left: 0,
+    zIndex: 1,
+  },
```

Comment:
If we use these classes without the `stickyElement` class, element won't be sticky. Should we rename these classes? (For example `headerTable` and `scaleTable` or something like that)